hack/devtools/rp-dev-helper.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+######## Helper file to run full RP dev either locally or using Azure DevOps Pipelines ########
+
+# Function to extract the image tag - (FUNCTION_NAME)
+extract_image_tag() {
```

Comment:
>Scraping the tags from const.go with grep would break if those functions or the patterns get changed, renamed or removed.

Indeed, it would require some maintenance and extra care for this changes in const.go. I am not sure how different it is then a possible change to Makefile targets that the suggested pipeline will use (e.g., `deploy` and `secrets`).

Reply:
> For the sake of simplicity, lets do this as a follow-up task. I will raise a jira to review this, if we don't have one already.

https://issues.redhat.com/browse/ARO-9238 👍🏻